### PR TITLE
IMX Support #6

### PR DESCRIPTION
**Pull request by clementfaure**
Hello,
Here is a small PR to fix minor issues on i.MX.

Thanks,
Clement

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: core/arch/arm/plat-imx/registers/imx6-crm.h [hunk 1]
```diff
@@ -2,8 +2,8 @@
 /*
  * Copyright 2017-2019 NXP
  */
-#ifndef __MX6_CRM_REGS_H__
-#define __MX6_CRM_REGS_H__
+#ifndef __MX6_CRM_H__
+#define __MX6_CRM_H__
```
Comment (jforissier) [inline core/arch/arm/plat-imx/registers/imx6-crm.h]: Nitpicking: should be __IMX6_CRM_H__ to better reflect the name of the file, and same below, but TBH we are not always consistent in the way we write those file guards :-/

Comment (clementfaure) [reply core/arch/arm/plat-imx/registers/imx6-crm.h]: You're right. I will fix this.

**Comment by jforissier** [comment]
@clementfaure feel free to squash the fixup and apply the Acked-by tags, this doesn't need more review.

**Comment by clementfaure** [comment]
Alright! Commits squashed and tags applied!

**Comment by jforissier** [comment]
The CI error needs to be addressed.

**Comment by clementfaure** [comment]
Oops! Warning fixed in the last commit.

**Comment by jforissier** [comment]
@clementfaure ha! that's pretty clever ;-) The obvious solution would have been to add the `UL` suffix except that it doesn't work since the value is used in the linker script (and perhaps in assembler too).
Personally, I can live with the hack.

`Acked-by: Jerome Forissier <jerome@forissier.org>`

**Comment by jenswi-linaro** [comment]
Perhaps `UINTPTR_C(0x60000000)` or some better suited macro to handle physical addresses would work?

**Comment by jforissier** [comment]
@jenswi-linaro good idea, I think something like this should work:
```C
#ifdef __ASSSEMBLER__
#define UL(x) x
#else
#define UL(x) x##UL
#endif
``` 

...because `__ASSEMBLER__` is defined when linker scripts are pre-processed.

@clementfaure care to try?

**Comment by clementfaure** [comment]
@jenswi-linaro That's smart!
@jforissier Sure! It works! Please check the last two commits.

**Comment by jforissier** [comment]
LGTM, thanks for trying this solution. Unless @jenswi-linaro has some comments, you may squash the commits.

**Comment by jenswi-linaro** [comment]
Looks good to me.

**Comment by clementfaure** [comment]
Do you ack 0cd7688239d544e43e6e8297ef41fe9aba61a13b then?

**Comment by jforissier** [comment]
@clementfaure 

> No problem! Do you ack 0cd7688239d544e43e6e8297ef41fe9aba61a13b then?

An informal acknowledgement such as "looks good to me" can generally be interpreted as a `Acked-by` tag especially if it's clear in the context of the discussion. So you may add Jens' tag to the above commit, as well as mine.

**Comment by clementfaure** [comment]
I'll keep that in mind!
Commits squashed and tags applied

**Pull request merged**